source/config/configSpec.py

```diff
@@ -45,6 +45,7 @@
 	autoDialectSwitching = boolean(default=false)
 	delayedCharacterDescriptions = boolean(default=false)
 	excludedSpeechModes = int_list(default=list())
+	_hasSapi4WarningBeenShown = boolean(default=False)
```

Comment:
Rather than tweaking the config spec, could you please consider adding a boolean setting to SupportedSettings on the sapi driver instead? I know that these settings show in the GUI by default, but that could easily be fixed by adding an extra showInGui parameter to DriverSetting. Or may be even better, just don't show all driver settings that are underscore prefixed and thus private to the synth.